### PR TITLE
feat(c3pm): convert test to ginkgo + resolve 1 test (2 errors still h…

### LINKED ISSUE #18: correct tests + convert existing tests to ginkgo

### PR DESCRIPTION
**Pull request by MinaBarry**
…ere)

### CODE REVIEW

File: cmakegen/cmakegen_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+)
+
+func TestCmakeGen(t *testing.T) {
+	RegisterFailHandler(Fail)
+	path, err := filepath.Abs("testsArtifacts")
+	fmt.Println("PATH:", path)
+	if err != nil {
+		t.Fatal("Failed to get testsArtifacts absolute path")
+	}
+	RunSpecs(t, "Registery Suite")
```
Comment (Oursin) [inline cmakegen/cmakegen_suite_test.go]: Here it should be "CMakeGen Suite" to reflect which suite it is

**Review by Oursin — APPROVED** [review]

File: cmakegen/cmakegen_suite_test.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package cmakegen
+
+import (
+	"fmt"
+	"os"
+
+	//"os"
```
Comment (Codelax) [inline cmakegen/cmakegen_suite_test.go]: ?

File: config/manifest/manifest_suite_test.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package manifest
+
+import (
+	"fmt"
+	"os"
+
+	//"os"
```
Comment (Codelax) [inline config/manifest/manifest_suite_test.go]: ?

File: registry/registry_suite_test.go [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package registry
+
+import (
+	"fmt"
+	"os"
+
+	//"os"
```
Comment (Codelax) [inline registry/registry_suite_test.go]: ?

**Review by Codelax — APPROVED** [review]

**Pull request merged**